editor/src/Components/DataTable.js

```diff
@@ -158,8 +171,10 @@ const DataTable = ({
                                         autoFocus
                                         name={field}
                                         id={field}
-                                        onChange={e => handleInputChange(e)}
-                                        defaultValue={fieldValue}
+                                        onChange={e =>
+                                          setEditValue(e.target.value)
+                                        }
+                                        value={editValue}
                                         type="select"
                                       >
                                         {selectOptions.map(option => (
```

Comment:
i just noticed we have an eslint error here because each <option> needs a unique key. would you mind setting the `key` prop on these 👇 ?